app/controllers/posts_controller.rb

```diff
@@ -22,7 +22,7 @@ class PostsController < ApplicationController
   end
 
   rescue_from Diaspora::NotMine do
-    render text: "You are not allowed to do that", status: 403
+    render text: I18n.t('posts.show.forbitten'), status: 403
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.